src/js/core/widget/core/VirtualListview.js

```diff
@@ -652,6 +661,10 @@
 				return result;
 			}
 
+			function fnRemoveCanvas(element) {
```

Comment:
I think that different name for instance: "isNotCanvas" or "isListItem" would suit better,
Also please consider changing to opposite logic and return true for elements such as <li> or <ol>

Reply:
Thanks. Done.